Authenticator/Classes/OTPAppDelegate.swift

```diff
@@ -25,6 +25,8 @@
 import UIKit
 import OneTimePassword
 
+let Z = "Z"
```

Comment:
Name Format Violation: Variable name should start with a lowercase character: 'Z'